config/locales/en.yml

```diff
@@ -311,6 +311,120 @@ en:
+        metadata associated with your course.</li>
+        <li>For each topic, click in the 'Add a new topic' box and begin to type its name. Select the correct topic.</li> 
+        <li>Repeat step 3 to add multiple topics.</li>
+        <li>You can reorder the topics using the arrow icons.</li>
+        <li>The learning path will be public by default. Untick the 'Public' box if you want to hide this learning path 
+        from anyone who isn&raquo;t the creator or a collaborator.</li>
+        <li>When you have finished, click the 'Register learning path' button.</li></ol>
+      collaborators: >
+        <p>Collaborators are TeSS users who can edit a learning path or topic after it has been created. 
+        Only the learning path or topic owner can add and remove collaborators. To add a collaborator:</p>
+        <ol><li>Navigate to the page for that learning path or topic.</li>
+        <li>Click the 'Collaborators' button at the top-right of the page.</li> 
+        <li>Start to type the name or username of the collaborator you wish to add and select them. 
+        The collaborator needs to be  a registered user of TeSS.</li>
+        <li>Repeat for any additional collaborators.</li>
+        <li>When you have finished, click the 'Done' button.</li></ol>
+      updating_paths: >
+        <p>To update a learning path or topic, you need to be the owner or a collaborator:</p>
+        <ol><li>Navigate to the page for that learning path or topic.</li>
+        <li>Click the 'Edit' button at the top-right of the page.</li> 
+        <li>Make your changes then click the 'Update learning path' button at the bottom.</li></ol>
+        <p>If you wish to delete the learning path or topic, use the 'Delete' button next to the 'Edit' button 
+        on the page for that item.</p>
+      view_paths: >
+        <p>Learning paths can be browsed and searched in TeSS. They are organised by topic.</p>
+        <ol><li>To view the learning paths, click 'Learning paths' in the main menu. 
+        (Tip: To view a learning path you do not need to be logged in to TeSS.)</li> 
+        <li>To browse all topics, click the 'Topics' button on the top-right.</li>
+        <li>Alternatively, you can click in the search box, type your query and press Enter.</li>
+        <li>Click to view a learning path. Read the summary box. The details of the included training materials 
+        appear below; click each item to open it. Information about the content provider appears on the left.</li> 
+        <li>To create a personal collection of your favourite learning paths, click the star icon at the top of each 
+        learning path to add them to a list. You can view your starred resources later by clicking the user icon 
+        at the top-right of the screen then 'My stars'.  To use the star feature you need to be a registered TeSS user.</li>
+      view_topics: >
+        <ol><li>On the 'Learning paths' tab, click the 'Topics' button at the top-right.</li>
```

Comment:
Forgot to mention in the original doc, but this button is actually labelled "Manage Topics" in the latest version of TeSS (not deployed yet)